CarRentals/CarRentalsWebAPI/Interfaces/IDataProcessing.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Collections.Generic;
+
+namespace CarRentals.Interfaces
+{
+    public interface IDataProcessing<T>
```

Comment:
we may want to add a GetAll() method too, what do you think?

Reply:
Added. I forgot to add this method to the interface.